### PR TITLE
Add documentation for error code 41.

### PR DESCRIPTION
**Pull request by internettrans**
See related https://github.com/single-spa/single-spa/pull/577. This should only be merged if that PR is merged.

### CODE REVIEW

File: website/src/components/Errors/Codes/41.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default function ErrorCode41(props) {
+  return (
+    <>
+      <h1>#41: single-spa is loaded on the page multiple times</h1>
+      <p>
+        The single-spa library should be loaded only one time on the page. This is important both for performance and to avoid bugs.
+        Any imported APIs
```
Comment (frehner) [inline website/src/components/Errors/Codes/41.js]: leftover line?

File: website/src/components/Errors/Codes/41.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <>
+      <h1>#41: single-spa is loaded on the page multiple times</h1>
+      <p>
+        The single-spa library should be loaded only one time on the page. This is important both for performance and to avoid bugs.
+        Any imported APIs 
+      </p>
+      <h2>To fix:</h2>
+      <p>
+        Mark single-spa as a <a href="https://webpack.js.org/configuration/externals/#root" rel="noopener">webpack external</a> in your root config and in each of your microfrontends.
```
Comment (frehner) [inline website/src/components/Errors/Codes/41.js]: did you mean to add `target="_blank"` to this anchor?

Comment (internettrans) [reply website/src/components/Errors/Codes/41.js]: I was going back and forth on it after having read a criticism of target blank ha. But yes I'll add it, good catch.

File: website/src/components/Errors/Codes/41.js (hunk 1 above)
Comment (internettrans) [reply website/src/components/Errors/Codes/41.js]: Just forgot to finish it.

**Review by frehner — APPROVED** [review]

**Pull request merged**